docs/sources/http_api/settings.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Settings API"
+description = "Enterprise Settings HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "settings", "enterprise"]
+aliases = ["/docs/grafana/latest/http_api/settings/"]
++++
+
+# Enterprise Settings API
+
+Settings API is only available in Grafana Enterprise. Read more about [Grafana Enterprise]({{< relref "../enterprise" >}}).
+
+## Update / remove database settings
+
+> **Note:** Available in Grafana Enterprise v8.0+.
+
+`PUT /api/admin/settings`
```

Comment:
Should we move it into the [`Admin API`](https://grafana.com/docs/grafana/latest/http_api/admin/) page? 🤔 

In fact, it's part of the Admin API (`/admin/*`), but it's only available in Enterprise, while all the other endpoints that belong to that API are available in OSS as well. Otherwise, we could rename it into something like Enterprise Admin API.

WDYT? cc/ @xlson

Reply:
I'll pull down the branch and take a look :)

Reply:
I think it makes sense to put it in the admin page as that's where all the other /admin/... endpoints are.

Reply:
Yeah, it does make sense! Thanks for your feedback! 🙏🏻